package.json

```diff
@@ -70,6 +71,16 @@
     "tsc-watch": "^4.4.0"
   },
   "resolutions": {
+    "@polkadot/api": "^5.0.1",
+    "@polkadot/api-contract": "^5.0.1",
+    "@polkadot/hw-ledger": "^7.0.1",
+    "@polkadot/keyring": "^7.0.1",
+    "@polkadot/networks": "^7.0.1",
+    "@polkadot/phishing": "^0.6.222",
+    "@polkadot/types": "^5.0.1",
+    "@polkadot/util": "^7.0.1",
+    "@polkadot/util-crypto": "^7.0.1",
+    "@polkadot/wasm-crypto": "^4.1.2",
```

Comment:
so if I recall, these will force the packaegs to be the versions listed.

What happens if two incompatible versions of a package are used throughout? I'd worry that this will lead to the thing expecting a different-enough version hitting errors and such?

Reply:
So with our resolutions, we are grabbing them directly from the resolutions from @polkadot. That way they will never be mismatched.

Reply:
Should we update the release readme to add this step to make sure that they remain in sync?

Reply:
(I haven't thought through the other ramifications of doing this yet; it makes me feel a bit uneasy offhand, but I'm sure it's been given some thought!)

Reply:
It's kinda a double edged sword too, because without this we potentially let multiple versions live inside of the node_modules which could also cause issue's. I shared the same concern as you, but after some digging I realized its probably best to mimic and shadow what `@polkadot` is doing which will atleast minimize the package conflictions that can potentially happen.

Reply:
> Should we update the release readme to add this step to make sure that they remain in sync?

Yes thank you for the reminder this should be updated in the README.md

Reply:
Ok cool! Does the readme want tweaking as part of this PR too or are you happy doing that in a next step?

Reply:
Yea just added the little bit on updating the resolutions, so you should be good :)

Reply:
Woop, thanks! Looks good!